tools/generate_vsn.sh

```diff
@@ -1,9 +1,48 @@
 #!/bin/bash
+
```

Comment:
I already suggested that some time ago and I'm going to repeat myself :) I think using at least

```bash
set -euo
```

will help to write better and easier to debug bash scripts. More details on the above and other flags can be found here: http://redsymbol.net/articles/unofficial-bash-strict-mode/

Reply:
Thanks. I haven't updated any shell scripts in repo for quite a while so I've simply forgotten about this. :)